Modules/IO/GDCM/test/CMakeLists.txt

```diff
@@ -185,6 +185,17 @@ itk_add_test(NAME itkGDCMImageReadWriteTest_JPEGBaseline1
       ${ITK_TEST_OUTPUT_DIR}/itkGDCMImageReadWriteTest_JPEGBaseline1.mha
       rgb)
 
+itk_add_test(NAME itkGDCMImageIOTest3_mono1
+  COMMAND ITKIOGDCMTestDriver
+    --ignoreInputInformation
+    --compare DATA{Baseline/itkGDCMImageIOTest3Baseline.mha}
+      ${ITK_TEST_OUTPUT_DIR}/itkGDCMImageIOTest3_mono1.mha
```

Comment:
@dzenanz Technically `itkGDCMImageIOTest3_mono1.mha` should have the same sha1 as `itkGDCMImageIOTest3.mha` so you could save some disk space.

Reply:
`itkGDCMImageIOTest3Baseline.mha` already exists, so I am reusing it. But orientation is changed, as well as origin. So that's why I needed to add `--ignoreInputInformation`.